Podfile

```diff
@@ -10,7 +10,7 @@ abstract_target 'Demo' do
 #  pod 'ChartboostMediationAdapterAdMob'
 #  pod 'ChartboostMediationAdapterAppLovin'
 #  pod 'ChartboostMediationAdapterAmazonPublisherServices'
-  pod 'ChartboostMediationAdapterChartboost'
+#  pod 'ChartboostMediationAdapterChartboost' # TODO: uncomment this for Mediation 5 when it's ready
```

Comment:
Temporarily disable this because current Demo app setup does not capture the latest updates in the adapters